src/components/views/rooms/RoomBreadcrumbs.js

```diff
@@ -81,6 +96,31 @@ export default class RoomBreadcrumbs extends React.Component {
         }
     }
 
+    onMyMembership = (room, membership) => {
+        if (membership === "leave" || membership === "ban") {
+            // Force left rooms to render appropriately
+            this.forceUpdate();
+        }
+    };
+
+    onRoomReceipt = (event, room) => {
+        if (this.state.rooms.map(r => r.room.roomId).includes(room.roomId)) {
+            this.forceUpdate();
```

Comment:
Can we put this in the state please and avoid `forceUpdate`? I know we do it elsewhere like this, but we should probably stop doing it :)